Dockerfile

```diff
@@ -1,4 +1,10 @@
+FROM gradle:7.0.2-jdk11 AS build
+COPY --chown=gradle:gradle . /home/gradle/src
+WORKDIR /home/gradle/src
+RUN gradle build --no-daemon 
+
 FROM openjdk:11
 EXPOSE 8080
-ADD build/libs/chat-0.0.1-SNAPSHOT.jar app.jar
-ENTRYPOINT ["java","-jar","/app.jar"]
+RUN mkdir /app
+COPY --from=build /home/gradle/src/build/libs/*.jar /app/spring-boot-application.jar
+ENTRYPOINT ["java","-jar","/app/spring-boot-application.jar"]
```

Comment:
add empty line, pls setup IDE to add empty line in each file. Also checkout Reformat Code and Optimise imports before commit.